kedro/contrib/idempotent/idempotent_state_storage.py

```diff
@@ -3,39 +3,60 @@
 from uuid import uuid4
 from typing import List, Any
 
-from kedro.io.data_catalog import MemoryDataSet
+from kedro.io.data_catalog import MemoryDataSet, DataCatalog
+
+IDEMPOTENT_STATE_STORAGE_CATALOG_NAME = 'idempotent_state_storage'
```

Comment:
Move this into the `IdempotentStateStorage` class, so that we have easier programmatic access to it.